Coordinator/Coordinator/App/View/LoginView.swift

```diff
@@ -0,0 +1,116 @@
+        textField.layer.borderWidth = 0.5
+        textField.layer.cornerRadius = 9
+        textField.alpha = 0.5
+        return textField
+    }()
+
+    let authButton: UIButton = {
+        let button = UIButton()
+        button.setTitle("Auth", for: .normal)
+        button.setTitle("No", for: .disabled)
+        button.setTitleColor(.black, for: .disabled)
+        button.layer.cornerRadius = 8
+        button.addTarget(self, action: #selector(toFirstVC), for: .touchUpInside)
+        return button
+    }()
+
+    //MARK: - Internal Properties
+    var completion: (()->())?
+
+    //MARK: - Initializers
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupView()
+        setupLoginTextField()
+        setupPasswordTextField()
+        setupAuthButton()
+        bindViewModel()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setupView()
+        setupLoginTextField()
+        setupPasswordTextField()
+        setupAuthButton()
+        bindViewModel()
+    }
+
+    //MARK: - Private Methods
+    @objc private func toFirstVC() {
+        completion?()
+    }
+
+    private func setupLoginTextField() {
+        loginTextField.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(loginTextField)
+        NSLayoutConstraint.activate([
+        loginTextField.centerXAnchor.constraint(equalTo: centerXAnchor),
+            loginTextField.centerYAnchor.constraint(equalTo: layoutMarginsGuide.topAnchor, constant: 150),
+        loginTextField.widthAnchor.constraint(equalToConstant: 100),
+        loginTextField.heightAnchor.constraint(equalToConstant: 30)
+                                        ])
+    }
+
+    private func setupPasswordTextField() {
+        passwordTextField.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(passwordTextField)
+        NSLayoutConstraint.activate([
+            passwordTextField.centerXAnchor.constraint(equalTo: centerXAnchor),
+            passwordTextField.centerYAnchor.constraint(equalTo: loginTextField.topAnchor, constant: 50),
+            passwordTextField.widthAnchor.constraint(equalToConstant: 100),
+            passwordTextField.heightAnchor.constraint(equalToConstant: 30)
+                                        ])
+    }
+
+    private func setupAuthButton() {
+        authButton.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(authButton)
+        NSLayoutConstraint.activate([
+            authButton.centerXAnchor.constraint(equalTo: centerXAnchor),
+            authButton.centerYAnchor.constraint(equalTo: passwordTextField.topAnchor, constant: 250),
+            authButton.widthAnchor.constraint(equalToConstant: 100),
+            authButton.heightAnchor.constraint(equalToConstant: 30)
+                                        ])
+    }
+
+    private func setupView() {
+        backgroundColor = .white
+    }
+
+    private func bindViewModel() {
```

Comment:
```suggestion
    private func bind() {
```
ViewModel - откуда?)